internal/datacoord/garbage_collector.go

```diff
@@ -158,8 +163,28 @@ func (gc *garbageCollector) scan() {
 }
 
 func (gc *garbageCollector) clearEtcd() {
+	req := &internalpb.GetImportFailedSegmentIDsRequest{
+		Base: &commonpb.MsgBase{
+			MsgType: commonpb.MsgType_GetImportFailedSegmentIDs,
+		},
+	}
+	ctx := context.Background()
```

Comment:
Check the healthiness of rootcoord before sending RPC

Reply:
this may be checked at rootcoord server

Reply:
Take a look at: https://github.com/milvus-io/milvus/blob/f0b036a35ae17ada3101eebc6a200b5f621b7e7e/internal/rootcoord/root_coord.go#L216

Reply:
this method is private and can only be call at rootcoord package？

Reply:
OK I guess just checking `rcc != nil` for now

Reply:
ok